docs/admin/welcome/index.md

```diff
@@ -19,6 +19,15 @@ curl -L https://opencloud.eu/install | /bin/bash
 - Configuration – Customize and integrate with existing systems and storage
 - Maintenance – Learn how to update, backup and migrate
 
+## Documentation Versions
+
+Our documentation is versioned to ensure you always find the right information for your software version.
```

Comment:
put yourself in the shoes of the reader - first question would be "which software version do i have? idk.." --> i suggest to tell where to find it.